common/achievement/get.ts

```diff
@@ -69,6 +69,7 @@ const assembleAchievementData = async (userAchievements: User_achievement[], use
     const newAchievement = state === achievement_state.COMPLETED && !userAchievements[resultIndex].isSeen;
 
     return {
+        id: userAchievements[resultIndex].id,
```

Comment:
Wouldn't this result in an error if `resultIndex == null`?

Reply:
I removed result index since it made no sense.